SignalServiceKit/src/Network/API/TSNetworkManager.m

```diff
@@ -56,6 +60,17 @@ - (void)makeRequest:(TSRequest *)request
             failure:(void (^)(NSURLSessionDataTask *task, NSError *error))failureBlock
 {
     DDLogInfo(@"%@ Making request: %@", self.logTag, request);
+    if (!CurrentAppContext().isMainApp) {
+        // TODO: Discuss which of these requests to suppress.
+        if (![request isKindOfClass:[TSRecipientPrekeyRequest class]]
+            && ![request isKindOfClass:[TSSubmitMessageRequest class]]
+            && ![request isKindOfClass:[OWSGetProfileRequest class]]
```

Comment:
per recent changes, I don't think we should be making profile requests either.

Reply:
Removed.